docs-src/typescript/how-to-run-a-worker-on-docker.md

```diff
@@ -0,0 +1,114 @@
+---
+id: how-to-run-a-worker-on-docker
+title: How to run a Worker on Docker in TypeScript
+sidebar_label: Run a Worker on Docker
+description: Workers based on the TypeScript SDK can be deployed and run as Docker containers.
+tags:
+  - developer-guide
+  - sdk
+  - typescript
+---
+
+Workers based on the TypeScript SDK can be deployed and run as Docker containers.
+
+At this moment, we recommend usage of Node.js 16.
```

Comment:
No more known issue on Node 18, but known issues for Node 20. Would recommend Node 18 at this point.

Reply:
Can update 16 => 18 on all Docker snippets in the document.